background.ts

```diff
@@ -922,7 +922,7 @@ ipcMainProxy.handle('versions/macOs', () => {
 });
 
 ipcMainProxy.handle('host/isArm', () => {
-  return Electron.app.runningUnderARM64Translation || process.arch.startsWith('arm');
+  return process.arch.startsWith('arm');
```

Comment:
Everywhere else we only check for `arm64`, so shouldn't it be the same here?
```suggestion
  return process.arch === 'arm64';
```

Reply:
Right, amended.